src/bacnet/datalink/bvlc6.h

```diff
@@ -12,8 +12,10 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <stddef.h>
+#include "bacnet/config.h"
```

Comment:
config.h is included in bacdef.h as the first define is is only included there. Please remove config.h here.

Reply:
Hmmmm.... It was missing the HOST_N_PORT datatype - shall take a second look. Thank you for the hint.

Reply:
done.